src/providers/http_provider.py

```diff
@@ -139,7 +141,7 @@ def _get_without_fallbacks(
                     code=0,
                     domain=urlparse(host).netloc,
                 )
-                raise error
+                raise self.PROVIDER_EXCEPTION(status=0, text='Response error.') from error
```

Comment:
does the final message in log include text from `error`?

Reply:
Will be something like this
```
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
  File "<stdin>", line 2, in func
ConnectionError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<stdin>", line 4, in <module>
RuntimeError: Failed to open database
```

Reply:
Yes, but it will be caught on the higher level in try\catch and wrapped by logger
<img width="818" alt="image" src="https://github.com/lidofinance/lido-oracle/assets/32727352/c822b021-cf7e-4800-8916-f8371294c4ef">

Reply:
Yeah, thats an issue